### PR TITLE
Chore - fix misspellings

### PR DESCRIPTION
**Pull request by clickthisnick**
### What does this PR do?
Fixes some misspelled words

### What issues does this PR fix or reference?
N/A

### New Behavior
Some comments will now have words spelled correctly.
Could potentially impact for better or worse alerts on logs.

### Tests written?
No

### Commits signed with GPG?
No

### CODE REVIEW

**Comment by waynew** [comment]
re-run all

File: salt/cloud/clouds/lxc.py [hunk 1]
```diff
@@ -232,7 +232,7 @@ def _salt(fun, *args, **kw):
                 ret = cret['data']
                 break
             # special case, some answers may be crafted
-            # to handle the unresponsivness of a specific command
+            # to handle the unresponsive of a specific command
```
Comment (waynew) [inline salt/cloud/clouds/lxc.py]: Not the right fix here.

File: salt/modules/glance.py [hunk 2]
```diff
@@ -168,7 +168,7 @@ def get(key, default=None):
                   ks_endpoint, kwargs)
         keystone = kstone.Client(**kwargs)
         kwargs['token'] = keystone.get_token(keystone.session)
-        # This doesn't realy prevent the password to show up
+        # This doesn't really prevent the password to show up
```
Comment (waynew) [inline salt/modules/glance.py]: Should be `really prevent the password from showing up`

File: salt/modules/heat.py [hunk 3]
```diff
@@ -155,7 +155,7 @@ def get(key, default=None):
 
     token = __salt__['keystone.token_get'](profile)
     kwargs['token'] = token['id']
-    # This doesn't realy prevent the password to show up
+    # This doesn't really prevent the password to show up
```
Comment (waynew) [inline salt/modules/heat.py]: Same here

File: salt/states/btrfs.py [hunk 4]
```diff
@@ -170,7 +170,7 @@ def subvolume_created(name, device, qgroupids=None, set_default=False,
 
     # Resolve first the test case. The check is not complete, but at
     # least we will report if a subvolume needs to be created. Can
-    # happend that the subvolume is there, but we also need to set it
+    # happened that the subvolume is there, but we also need to set it
```
Comment (waynew) [inline salt/states/btrfs.py]: Should be `happen`

File: salt/states/modjk_worker.py [hunk 5]
```diff
@@ -106,7 +106,7 @@ def _worker_status(target,
         target, 'modjk.worker_status', args, tgt_type
     )
 
-    # Did we got any respone from someone ?
+    # Did we get any response from someone ?
```
Comment (waynew) [inline salt/states/modjk_worker.py]: Probably should read `Did we get a response from anyone?`

**Review by waynew — CHANGES_REQUESTED** [review]

**Comment by waynew** [comment]
@clickthisnick there are a couple of fixes to address here, are you still interested in getting this in?

**Comment by sagetherage** [comment]
@clickthisnick can you please rebase this against the master branch?

**Comment by sagetherage** [comment]
Reviewed in Grooming this week considering closing

**Pull request closed without merging**